libs/alac/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2016 OpenWrt.org  
+#
+# This is free software, licensed under the GNU General Public License v2 and the Apple Public Source License v2.0
+
+# Please see LICENSE and APPLE_LICENSE.txt for more information.
+# 
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=alac
+PKG_VERSION:=0.0.7
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/mikebrady/alac
```

Comment:
Please use the official release or repository as a source and add patches if needed.